annotation/util.scm

```diff
@@ -25,6 +25,7 @@
 	#:use-module (opencog exec)
 	#:use-module (opencog bioscience)
 	#:use-module (annotation graph)
+  #:use-module (opencog grpc)
```

Comment:
No code for module (opencog grpc)

Reply:
Fixed the above issues and run make check 
PASS: tests/parser-tests.scm
PASS: tests/util-tests.scm
PASS: tests/pathway-tests.scm

Prints the above and takes forever, it didnt finish

I run each tests on its own and 
go-tests failed 
- (find-go-plus (ConceptNode "GO:1903896")) returns empty
main test failed at (length (parse-request (list "IGF1") "Dfaer" req))
- Wrong number of arguments to #<procedure parse-request (req)
...

Reply:
Removed the grpc module